Superbuild/ITKExternal.cmake

```diff
@@ -50,7 +50,9 @@ SET(itk_CACHE_ARGS
   "-DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}"
   "-DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>"
   "-DITK_BUILD_DEFAULT_MODULES:BOOL=OFF"
-  "-DITKV3_COMPATIBILITY:BOOL=OFF"
+  "-DITKV4_COMPATIBILITY:BOOL=ON"
```

Comment:
The combination of settings here is indicative of full update to ITKv4, and minimal update to ITKv5. For future proofing, we should [finish migration to ITKv5](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Documentation/ITK5MigrationGuide.md#legacy-code-removed). That should be done in a new PR. Possibly after allowing a week or so of use to see if there are any immediate bugs/changes caused by this PR.

Reply:
Turning on removal of legacy code did not require further changes on my machine. I created an alternative PR with full commit history in #382.